Rakefile

```
@@ -3,13 +3,13 @@
 begin
   require 'rspec/core/rake_task'
   RSpec::Core::RakeTask.new(:spec)
-rescue LoadError # rubocop:disable Lint/HandleExceptions
+rescue LoadError
 end
 
 begin
   require 'rubocop/rake_task'
   RuboCop::RakeTask.new(:rubocop)
-rescue LoadError # rubocop:disable Lint/HandleExceptions
+rescue LoadError
```

Do not suppress exceptions.